manager/src/main/java/inventory/manager/Services/IProductController.java

```diff
@@ -0,0 +1,4 @@
+package inventory.manager.Services;
+
+public interface IProductController {
```

Comment:
Is this supposed to be IProductService? There is already a ProductController and it is not an interface.